packages/common/src/compiler/translator.ts

```diff
@@ -296,18 +299,26 @@ export const convertTable = (
                 startOfWeek,
             );
 
-            let extraDimensions = {};
-
-            const processIntervalDimension = (dim: Dimension) => {
-                if (isInterval(dim.type, column)) {
+            const processIntervalDimension = (
+                dim: Dimension,
+                overrideTimeIntervals: DbtColumnLightdashDimension['time_intervals'],
+            ) => {
+                if (dim.isIntervalBase) {
```

Comment:
This fixes the issue where when a `additional_dimension` had `time_intervals: "OFF"` then it would still show as both `raw` value + default time frames. 

A dimension with `time_intervals: "OFF"` is NOT `isIntervalBase` and therefore no intervals are generated.